app/models/form_item.rb

```diff
@@ -156,6 +159,14 @@ def display_conditionally?
     display_if != "always" && display_conditions.any?
   end
 
+  def condition_group
+    @condition_group ||= Forms::ConditionGroup.new(true_if: display_if, members: display_conditions)
+  end
+
+  def computed_condition_group
```

Comment:
We ended up not using this method, right, due to performance concerns? And the delegator above? I think we should remove it because it's too tempting.